src/message_wrapper.cpp

```diff
@@ -864,21 +879,10 @@ const sbg_driver::msg::SbgImuData MessageWrapper::createSbgImuDataMessage(const
 
   if (m_use_enu_)
   {
-    imu_data_message.accel.x        = ref_log_imu_data.accelerometers[0];
-    imu_data_message.accel.y        = -ref_log_imu_data.accelerometers[1];
-    imu_data_message.accel.z        = -ref_log_imu_data.accelerometers[2];
-
-    imu_data_message.gyro.x         = ref_log_imu_data.gyroscopes[0];
-    imu_data_message.gyro.y         = -ref_log_imu_data.gyroscopes[1];
-    imu_data_message.gyro.z         = -ref_log_imu_data.gyroscopes[2];
-
-    imu_data_message.delta_vel.x    = ref_log_imu_data.deltaVelocity[0];
-    imu_data_message.delta_vel.y    = -ref_log_imu_data.deltaVelocity[1];
-    imu_data_message.delta_vel.z    = -ref_log_imu_data.deltaVelocity[2];
-
-    imu_data_message.delta_angle.x  = ref_log_imu_data.deltaAngle[0];
-    imu_data_message.delta_angle.y  = -ref_log_imu_data.deltaAngle[1];
-    imu_data_message.delta_angle.z  = -ref_log_imu_data.deltaAngle[2];
+    imu_data_message.accel          = convertNedArrayToEnuVector(ref_log_imu_data.accelerometers);
```

Comment:
IMU measurements are in body frame not in NED.
We should thus apply the following conventions:
X = X
Y = -Y
Z = -Z